ui/apps/ui/src/app/components/widgets/upcoming-events.component.ts

```diff
@@ -0,0 +1,168 @@
+                >25-26.11.2022 Paris, France</span
+              >
+            </div>
+            <div class="row">
+              <span class="widget-header-theme">Final Conference</span>
+            </div>
+            <div class="row pt-3">
+              <div class="col-6">
+                <div (click)="interested()" class="interested">
+                  <span class="interested-text">Interested?</span
+                  ><img src="assets/interested.svg" />
+                </div>
+              </div>
+              <div class="col-4">
+                <div><span class="going-text">36 Going</span></div>
+              </div>
+            </div>
+            <div class="row" align="end">
+              <span>
+                <button
+                  [disabled]="!isNext()"
+                  type="button"
+                  class="btn px-0 py-0"
+                  (click)="getNext()"
+                >
+                  <img width="24px" height="24px" src="assets/left_icon.svg" />
+                </button>
+                <button
+                  [disabled]="!isPrev()"
+                  type="button"
+                  class="btn px-0 py-0"
+                  (click)="getPrev()"
+                >
+                  <img width="24px" height="24px" src="assets/right_icon.svg" />
+                </button>
+              </span>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  `,
+  styles: [
+    `
+      .widget-header-place {
+        font-family: 'Switzer';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 14px;
+        line-height: 26px;
+        color: #1a2128;
+      }
+      .widget-header-theme {
+        font-family: 'Switzer';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 20px;
+        line-height: 26px;
+        color: #1a2128;
+      }
+      .interested {
+        cursor: pointer;
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        padding: 6px 16px;
+        gap: 8px;
+        background: rgba(25, 117, 255, 0.08);
+        border-radius: 10px;
+      }
+      .interested-text {
+        font-family: 'Switzer';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 13px;
+        line-height: 18px;
+        color: #144b9e;
+      }
+      .going-text {
+        font-family: 'Switzer';
+        font-style: normal;
+        font-weight: 500;
+        font-size: 14px;
+        line-height: 18px;
+        color: #919ba2;
+      }
+      #show-more {
+        padding-right: 6px;
+      }
+    `,
+  ],
+})
+export class WidgetUpcomingEventsComponent implements OnInit {
+  backendUrl = `${environment.backendApiPath}`;
+  // TODO To be replaced by Interface and Service
+  upcomingEvents!: Observable<
+    {
+      id: string;
+      img: string;
+      date: string;
+      theme: string;
+      going: number;
+    }[]
+  >;
```

Comment:
Instead of the code create two new files: 
- `events.service.ts` as a regular Angular service with one method `get$` that will fetch data from external sources and initially return `of([])`
https://angular.io/guide/architecture-services#service-examples

- `events.interface.ts` as incoming data interface with beneath content

```javascript
{
      id: string;
      img: string;
      date: string;
      theme: string;
      going: number;
    }
```